pkg/amazon/backend/cloudformation.go

```diff
@@ -218,9 +220,18 @@ func getLoadBalancerSecurityGroups(project *types.Project, template *cloudformat
 }
 
 func createLoadBalancer(project *types.Project, template *cloudformation.Template) string {
+	ports := 0
+	for _, service := range project.Services {
+		ports += len(service.Ports)
+	}
+	if ports == 0 {
+		// Project do not expose any port (batch jobs?)
+		// So no need to create a LoadBalancer
+		return ""
+	}
 
 	// load balancer names are limited to 32 characters total
-	loadBalancerName := fmt.Sprintf("%.32s", fmt.Sprintf("%sLoadBalancer", strings.Title(project.Name)))
```

Comment:
Are we dealing with the 32 char limit somewhere else now?

Reply:
my bad, merge resolution mistake

Reply:
got my working copy messed with git remote URL and eventually removed remote branch ...
starting clean on another PR